### ISSUE TITLE
Deprecated Gradle

### ISSUE THREAD

**Issue by miton18** [senx/warp10-module-template#8]
I have an issue at first gradlew run

```
gradlew bootstrap -Pg=io -Pa=auth.plugin -Pv=0.0.1 -Pd='ddds'
Downloading https://services.gradle.org/distributions/gradle-4.10.2-all.zip
...............................................................................................................

Welcome to Gradle 4.10.2!

Here are the highlights of this release:
 - Incremental Java compilation by default
 - Periodic Gradle caches cleanup
 - Gradle Kotlin DSL 1.0-RC6
 - Nested included builds
 - SNAPSHOT plugin versions in the `plugins {}` block

For more details see https://docs.gradle.org/4.10.2/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Task 'system' not found in root project 'myartifact'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 30s
```

**Comment by hbs** [comment]
Since you seem to be running Windows, which we do not recommend, have you tried running gradlew.bat rather than gradlew?

**Comment by hbs** [comment]
Fixed by #9
